modules/dnn/src/layers/dummy_layer.cpp

```diff
@@ -0,0 +1,189 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+// Copyright (C) 2021, Intel Corporation, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
```

Comment:
Please remove this unrelated copyright. The first 3 lines from Wiki page is enough.

Reply:
Fixed